run.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 exec java ${JAVA_OPTS} \
+  -XX:+PrintCompilation -XX:TieredStopAtLevel=4 -XX:Tier3CompileThreshold=1 -XX:Tier4CompileThreshold=1 \
+  -XX:Tier3InvocationThreshold=1 -XX:Tier4InvocationThreshold=1 \
+  -XX:Tier3MinInvocationThreshold=1 -XX:Tier4MinInvocationThreshold=1 \
```

Comment:
This turns on almost immediate JIT compilation. The theory is that most will be complete by the time traffic is let into the pod. Any remaining compilation (in endpoint app level code) will be done much earlier resulting in very few invocations on interpreted code.